bin/node/runtime/src/lib.rs

```diff
@@ -128,7 +128,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	// and set impl_version to 0. If only runtime
 	// implementation changes and behavior does not, then leave spec_version as
 	// is and increment impl_version.
-	spec_version: 30500,
+	spec_version: 30510,
```

Comment:
Should become 30600 according to [spec_version format](https://github.com/Cerebellum-Network/pos-network-node/blob/638f2d82296ed5567d6e19dbe613adba87b60f75/CHANGELOG.md#changed-3).

Reply:
Thanks. Updated.